src/providers/chatwoot.ts

```diff
@@ -0,0 +1,55 @@
+import { State } from 'types'
+
+const domain = 'https://app.chatwoot.com'
+
+declare global {
+  interface Window {
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    chatwootSettings: any
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    $chatwoot: any
+    chatwootSDK: {
+      run: (options: { websiteToken: string; baseUrl: string }) => void
+    }
+  }
+}
+
+/* eslint-disable */
+const loadScript = (onload: () => void) => {
+  if (window.$chatwoot) return
+  ;(function(d, t) {
+    var script: HTMLScriptElement = d.createElement('script')
+    var fisrtScript = d.getElementsByTagName('script')[0]
+    script.src = domain + '/packs/js/sdk.js'
+    fisrtScript.parentNode?.insertBefore(script, fisrtScript)
+    script.onload = onload
+  })(document)
+}
+/* eslint:enable */
```

Comment:
I believe this should be `/* eslint-enable */` as per [the docs](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments). There were some incorrect references on main which I've addressed in another PR.